docs/reference-guides/cluster-configuration/rancher-server-configuration/sync-clusters.md

```diff
@@ -40,6 +40,10 @@ The AKSConfig, EKSConfig or GKEConfig represents the desired state. Nil values a
 
 UpstreamSpec represents the cluster as it is in the hosted Kubernetes provider. It's refreshed every 5 minutes. After the UpstreamSpec is refreshed, Rancher checks if the cluster has an update in progress. If it's currently updating, nothing further is done. If it is not currently updating, any managed fields on AKSConfig, EKSConfig or GKEConfig are overwritten with their corresponding value from the recently updated UpstreamSpec.
 
+:::warning
+When clusters are created via cloud provider and then imported into Rancher, UpstreamSpec represents the cluster state and Config is empty. If, after importing, the cluster is updated via Rancher UI, both specifications (UpstreamSpec and Config) are non-null and any further change should be performed via Rancher, as there is no safe way to identify that changes originating in UpstreamSpec represent the desired state or just a mismatch with Config. This means that, if updating the imported cluster from the cloud provider console after having applied any updates from Rancher UI, the controller will deploy a rollback and the content of Config will be considered as desired state.
```

Comment:
```suggestion
When you import a cluster from a cloud provider into Rancher, UpstreamSpec represents the cluster state and Config is empty. If you then update the imported cluster through the Rancher UI, both UpstreamSpec and Config become non-null. Any further updates to the cluster should be applied through Rancher. This is because there is no safe way to determine if changes originating from UpstreamSpec represent the desired state or just a mismatch with Config. If you update the imported cluster through the cloud provider console after you apply any updates through the Rancher UI, the controller will deploy a rollback and the content of Config will be considered the desired state.
```